tests/testthat/setup-bugRzilla.R

```diff
@@ -1,4 +1,5 @@
 library("vcr") # *Required* as vcr is set up on loading
+library("httptest") # *Required* as hhtptest is set up on loading
```

Comment:
The package is no longer used, please remove it from here and from the Description file.